app/assets/stylesheets/admin/admin.scss

```diff
@@ -0,0 +1,21 @@
+@import "../shared/color";
+@import "../shared/utilities";
+@import "../shared/markdown";
+
+.card .divider {
+  background-color: rgba(160, 160, 160, 0.2);
+}
+
+.form {
+  .event_short_description textarea {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `event_short_description` should be written in lowercase with hyphens